File: CS/WindowsFormsApplication475/MyColorEditViewInfo.cs
using DevExpress.Utils;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication475 {
    public class MyColorEditViewInfo : ColorEditViewInfo {
        public MyColorEditViewInfo(RepositoryItem item)
            : base(item) {

        }
protected override void CalcColorBoxRect() {
    fColorTextRect = MaskBoxRect;
    if (Item.ColorAlignment == HorzAlignment.Default) return;
    fColorBoxRect = Rectangle.Inflate(MaskBoxRect, 0, 0);
    fColorBoxRect.Width = MaskBoxRect.Width;
    fColorBoxRect.Height = MaskBoxRect.Height;
    if (Item.ColorAlignment == HorzAlignment.Far) fColorBoxRect.X = MaskBoxRect.Right - ColorIndent - fColorBoxRect.Width;
    if (Item.ColorAlignment == HorzAlignment.Center) fColorBoxRect.X = MaskBoxRect.Left + (MaskBoxRect.Width - fColorBoxRect.Width) / 2;
    //base.CalcColorBoxRect();
}
    }

}

File: VB/WindowsFormsApplication475/MyColorEditViewInfo.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Utils
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication475
	Public Class MyColorEditViewInfo
		Inherits ColorEditViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)

		End Sub
Protected Overrides Sub CalcColorBoxRect()
	fColorTextRect = MaskBoxRect
	If Item.ColorAlignment = HorzAlignment.Default Then
		Return
	End If
	fColorBoxRect = Rectangle.Inflate(MaskBoxRect, 0, 0)
	fColorBoxRect.Width = MaskBoxRect.Width
	fColorBoxRect.Height = MaskBoxRect.Height
	If Item.ColorAlignment = HorzAlignment.Far Then
		fColorBoxRect.X = MaskBoxRect.Right - ColorIndent - fColorBoxRect.Width
	End If
	If Item.ColorAlignment = HorzAlignment.Center Then
		fColorBoxRect.X = MaskBoxRect.Left + (MaskBoxRect.Width - fColorBoxRect.Width) / 2
	End If
	'base.CalcColorBoxRect();
End Sub
	End Class

End Namespace
